### ISSUE TITLE
Importing TestRPCProvider gives an error.

### ISSUE THREAD

**Issue by agatsoh** [pipermerriam/web3.py#64]
* Version: 1.6.0
* Python: 2.7
* OS: Ubuntu 14.04

### What was wrong?
Importing TestRPCProvider gives an error.
```
In [1]: from web3 import Web3, TestRPCProvider
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-92ad247d8f86> in <module>()
----> 1 from web3 import Web3, TestRPCProvider

ImportError: cannot import name TestRPCProvider
```

#### Cute Animal Picture

![Cute animal picture]()
![image](https://cloud.githubusercontent.com/assets/15123108/17669406/277a772a-632b-11e6-8bfe-4ba826f2d2f0.png)

**Comment by heikoheiko** [comment]
yes, i also ran into this. I think the readme is outdated.

**Comment by pipermerriam** [comment]
This is fixed in `1.7.1` and available on pypi or in the releases section of this repo. You can now import the `TestRPCProvider` directly from `web3`.
